The upstream commit 1e422ed of keyring moves non-preferred keyring
backends to keyrings.alt package, so porting chown_keyringlock_file.patch
to package keyrings.alt

Index: keyring-5.3/keyrings/alt/file_base.py
===================================================================
--- keyring-5.3.orig/keyrings/alt/file_base.py
+++ keyring-5.3/keyrings/alt/file_base.py
@@ -196,6 +196,7 @@ class Keyring(FileBacked, KeyringBackend
                  pass
              # must have the lock file with the correct group permissisions g+rw
              os.chmod(lockdir + "/" + lockfile, stat.S_IRWXG | stat.S_IRWXU)
+             os.chown(lockdir + "/" + lockfile,-1,345)
 
 
     def delete_password(self, service, username):
